packages/app/src/systems/Bridge/machines/bridgeMachine.tsx

```diff
@@ -120,7 +120,9 @@ export const bridgeMachine = createMachine(
         toNetwork: ev.input.toNetwork,
       })),
       assignAssetAmount: assign({
-        assetAmount: (_, ev) => ev.input.assetAmount,
+        assetAmount: (_, ev) => {
+          return ev.input.assetAmount?.eq(0) ? undefined : ev.input.assetAmount;
+        },
```

Comment:
just like the AmountInput talk we had, same thing applies here.

0 is 0, undefined is undefined, and also other values are possible. we should not put always undefined if it's zero.